nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/migrator/AbstractRepositoryMigrator.java

```diff
@@ -58,8 +58,19 @@ protected void reportProgress(String repositoryName, String message, long num, l
         reportProgress(String.format("[%s] %s", repositoryName, message), num, total);
     }
 
+    /**
+     * @deprecated since 11.1. Use {@link #migrateRepository(MigrationContext, String, String)} instead
+     */
+    @Deprecated(since = "11.1", forRemoval = true)
```

Comment:
There's also the field and its usage to deprecate. You will need to keep affectation in order to be backward compatible, as it you can use these deprecated field(s) in the deprecated method to provide backward compatibility.

Reply:
yep thx, we forgot  this one